CGPs/0017.md

```diff
@@ -105,13 +107,20 @@ Note:
     *   Less than $450/month in operational expenses 
 
 ## Proposed Changes
-1. Description of transaction (TBD)
-  - Destination: TBD
-  - Data: TBD
-  - Value: 665,387 CELO
+ERC-20 Approval of 665,387 * (10 ** 18) CELO to multsig wallet at 0x2f90Bb1CAF4079192481B1A7F3088220d291456D  
+Destination: GoldToken, approve  
+Data: 0x2f90Bb1CAF4079192481B1A7F3088220d291456D, 665387000000000000000000 (665387e18)  
+Value: 0  
+
+In this execution, the governance proposal is approving 665,387 CELO allowance of the community fund (the governance address) to the multisig address rather than transferring the CELO fund to the multisg contract itself. This method allows flexibility in removing the allowance later if needed and de-risks lost funds if ever the multisig became inaccessible.
+
+Multisig owners:  
+0x099f3F5527671594351E30B48ca822cc90778a11  
```

Comment:
One of the best practices is assigning the person's name and relation to CELO to the key that they control. They did that in the cUSD rewards CGP proposal, I think it's good practice to do it here.

Reply:
I will add the names of the holders here. the relation to Celo should be available in our bios below